app/controllers/SchemeController.scala

```diff
@@ -66,8 +70,11 @@ class SchemeController @Inject()(schemeService: SchemeService,
 
   def removePsa: Action[PsaToBeRemovedFromScheme] = Action.async(parse.json[PsaToBeRemovedFromScheme]) {
     implicit request =>
-      schemeConnector.removePSA(request.body)map {
-        case Right(_) =>  NoContent
+      val psaToBeRemoved: PsaToBeRemovedFromScheme = request.body
+      schemeConnector.removePSA(psaToBeRemoved) map {
+        case Right(_) =>
+          auditService.sendEvent(PSARemovalFromSchemeAuditEvent(psaToBeRemoved))
```

Comment:
can we move the audit logic to schemeConnector.removePSA

Reply:
Sorted.